src/cryptoadvance/specter/commands/utxo_scanner.py

```diff
@@ -0,0 +1,234 @@
+import threading
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class UtxoScanner:
+    """A Command class which simplifies the scanning of UTXOs and the import into a (core+specter) wallet"""
+
+    # 1. pass descriptor (change + recv) and range to rpc.scantxoutset
+    #    get list of tx each having deriv-path
+    # 2. Update the address-indexes with the max from recv+change (to get fresh addresses and not used once)
+    # 3. rpc.getblockhash of height(each tx)
+    # 4. rpc.gettxoutproof get a proof that the tx is included in a block, needs tx_id and blockhash for each tx
+    #    This will only work on fullnodes (!)
+
+    timeout = (
+        300  # 0.001 # As this is async, have longer timeout for all rpc-calls: 5 mins
+    )
+
+    def __init__(
+        self, wallet, explorer: str = None, requests_session=None, only_tor=False
+    ):
+        self.wallet = wallet
+        self.explorer = explorer
+        if self.explorer:
+            self.explorer = self.explorer.rstrip("/")
+        self.requests_session = requests_session
+        self.only_tor = only_tor
+        self.error_msgs = []
+
+    def execute(self, asyncc=False):
```

Comment:
asyncc with two cc?

Reply:
Yes, as async is a reserved word in python. I still don't have a consistent pattern how to deal with such cases. That's somehow i come up with `claz` or `clas`.